### ISSUE TITLE
Looking for help maintaining this project

### ISSUE THREAD

**Issue by bunkat** [bunkat/later#184]
Hi,

I'm really glad that so many people have found Later to be useful in their projects.  Unfortunately, I'm personally no longer using this library and don't have time to keep it properly maintained (obvious by the mounting issues and lack of responses, which I feel horrible about).

If you are currently using Later and would like to help maintain this library, please let me know!  

Thanks,
bill

**Comment by hovancik** [comment]
Hi, maybe put link to this somewhere on the first lines of readme?

**Comment by jagan-veeraraghavan** [comment]
Hi bunkat,
I would like to help maintain this library. Though this would be my first Open Source contribution.

**Comment by TimNZ** [comment]
@bunkat  I've been thinking for a while that Github should bake in a mechanism to make it easy to advertise owners wanting to find someone to take over a repo, yours being up there as a motivator.

I'm far from a JS expert but this is such a sweet library that is the core of a product I'm happy to help as a Collaborator.  There are some bugs that I struggled to trace through your code that I would have another go at as a starting point.

What are you working on instead?

**Comment by bunkat** [comment]
Thanks for the interest, sorry I didn't reply sooner.  I'd be glad to have the help.  If you are looking for a place to start, there are a bunch of outstanding pull requests, it would be great to go through those to see which ones are still valid, have test cases that pass and are ready to be merged.

I moved from NodeJS to Go and have since ported and optimized this library for my needs.  Since I'm no longer using this version, I really can't justify spending time maintaining it anymore.

**Comment by TimNZ** [comment]
Fair enough.

Realistically I think I'll be challenged time/priority wise in the short term as well.
I'll definitely make time in December.

**Comment by jagan-veeraraghavan** [comment]
Let me take up the outstanding pull requests over the weekends and get back. Shall we continue on this conversation in the same thread or is there any other collaboration tool that you prefer.

**Comment by QuantumInformation** [comment]
Are there any other npm libs that provide a way to validate a cron expression?

**Comment by rex** [comment]
@bunkat Are you still interested in finding maintainers for this library? I am a developer at @TopGolf and being able to bring this library under our development umbrella would be a fantastic win for us since we handle so many time-related operations in javascript. Please let me know if you might be interested so that I can get the necessary people involved in the conversation on our end!

**Comment by bunkat** [comment]
@rex Yeah, I would be open to that if your team is interested in maintaining the library.  Let me know what I can do to help that effort.

**Comment by alecl** [comment]
Anything come of this? Would be great for someone to take this on @rex.

**Comment by alecl** [comment]
@TimNZ @jagan-veeraraghavan Still interested too?

**Comment by TimNZ** [comment]
@alecl Yes, low level contributor at best. 

I did maintenance on an app using Later today, which reminded me there are things I want to fix and enhance.  

Same for http://bunkat.github.io/schedule/

**Comment by ghostsquad** [comment]
What's the status of migrating ownership?

**Comment by gabrielgrant** [comment]
@ghostsquad seems to be blocked on finding someone willing to commit to maintaining the lib. You offering? ;)

**Comment by ghostsquad** [comment]
@gabrielgrant unfortunately I'm not really a javascript developer. I dabble in it, but I wouldn't be a good candidate for ownership.

**Comment by kirkins** [comment]
Hi, my boss is asking me to make modifications to this library. I asked about using a different library as this one seems abandoned, but they want to keep using it.

I don't think I would be the perfect maintainer but if no else wants to. It has been a long time...

I have permission to use my paid time at work towards contributing to this project.

**Comment by kirkins** [comment]
I fixed the travis so it runs the test and sorting through the PRs now.

https://github.com/kirkins/later

I haven't actually used this library myself yet but my company needs to upgrade it.

We already used `later` in a lot of places but now we're having problems related to timezones and day-light savings. So after I merge the newish PRs I will start trying to find a solution to #95.

If anyone more experienced with the library wants to help with the whole process that would be great! I'm probably the wrong person to do it, but funner than other things I could be working on! 😊 ✌

**Comment by rramphal** [comment]
Hey @bunkat, I saw that back on Nov 8, 2016, you said you ported this library to Go. Do you have any plans on releasing the Go port? 😄

**Comment by bunkat** [comment]
@kirkins I can give you write access to this project if you wanted to take over maintenance / updates. Sorry I didn't get back to you sooner.

@rramphal No, the Go port is not a general purpose library like this one was.  It was purpose built to do a few scheduling operations very fast.

**Comment by kirkins** [comment]
@bunkat sure if you are OK with which PRs I brought in on my branch I can do the same over here :-)

**Comment by naz** [comment]
In case anybody comes around this thread looking for support. The most maintained and active fork for this project is https://github.com/breejs/later
